backend/resources/books.py

```diff
@@ -145,16 +146,15 @@ def get(self):
         parser.add_argument('score', type=bool, required=False,
                             help="Indicates if returning the score of the book is needed .")
         data = parser.parse_args()
+        if not data:
```

Comment:
No entrara main perque data mai estara buit, tindries que comprovar si els valors son diferents a `None`. A mes de comprovar que els valors que no son `None` no siguin els dels booleans.
```suggestion
        if not all(v is not None for k, v in data.items() if k not in ['reviews', 'score']):
```

Reply:
Cert ben vist. M'he despistat pq abans li feia un dict comprehension al data. 
No funciona amb all, ha de ser any. 

He afegit dos tests per comprovar que funciona adecuadament.